python-sdk/src/astro/databases/snowflake.py

```diff
@@ -638,6 +639,22 @@ def _copy_into_table_from_stage(self, source_file, target_table, stage):
             self.drop_stage(stage)
         return rows
 
+    def _validate_before_copy_into(self, source_file, target_table, stage):
+        """Validate COPY INTO command to tests the files for errors but does not load them."""
+        if self.load_options.validation_mode is None:
+            return
+        table_name = self.get_table_qualified_name(target_table)
+        file_path = os.path.basename(source_file.path) or ""
```

Comment:
@sunank200  if the file_path is empty, are we loading all the files in a folder?

Reply:
Yes it loads all the file path in a folder.